tests/Bridge/Laravel/Providers/LoyaltyCorpAuditingProviderTest.php

```diff
@@ -38,10 +41,56 @@
  */
 class LoyaltyCorpAuditingProviderTest extends TestCase
 {
+    /**
+     * Ensure middleware is applied on boot when appropriate
+     *
+     * @return void
+     */
+    public function testMiddlewareAppliedWhenEnabled(): void
+    {
+        /**
+         * Illuminate's packages have odd type hints
+         *
+         * @var \Tests\LoyaltyCorp\Auditing\Stubs\Vendor\Illuminate\ApplicationStub $application
+         * @var \Illuminate\Contracts\Foundation\Application $application
+         */
+        $application = new ApplicationStub();
+        \putenv('AUDITING_MIDDLEWARE=true');
```

Comment:
@sjdaws with modifying environment variables for tests, should I restore to the default value afterwards?

Reply:
In a test always use `setEnv` and `getEnv` so they're automatically cleaned up

Reply:
You should also add this value to phpunit.xml